sdk/servicebus/service-bus/src/core/autoLockRenewer.ts

```diff
@@ -185,7 +187,8 @@ export class LockRenewer {
                 bMessage.lockedUntilUtc = await this._context
                   .getManagementClient(linkEntity.entityPath)
                   .renewLock(lockToken, {
-                    associatedLinkName: linkEntity.name
+                    associatedLinkName: linkEntity.name,
+                    ...operationOptions
```

Comment:
I'm not sure if there's a convention on this but I try to do the `...`'s before my explicitly named properties (so in case there is overlap, "my" changes win). Since this is a runtime copy (ie, even stuff outside of the interface will be copied) that seems safer.

Reply:
I see, I can swap.